lib/manager/setup-cfg/__fixtures__/setup-cfg-1.txt

```diff
@@ -1,41 +1,44 @@
-foo >=1.1.1
-bar = bar >=2.2.2
 [options]
-baz >=3.3.3.
```

Comment:
The main question here is: even if those declarations are wrong, will they work?

Are they somehow deprecated? Them we should still support them, like we do on other managers too.

We should break users if this still works with wrong config.

Reply:
I've searched for some hours, but I didn't find anything about this style of config.
* `setuptools` didn't use this style.
* [`distutils2`](https://wiki.python.org/moin/Distutils2) no development since 2012. I didn't find no documentation for it.
* [`d2to1`](https://github.com/embray/d2to1) no development since 2015. Similar to `setuptools`. Marked as deprecated.
* [`pbr`](https://github.com/openstack/pbr) is mainly the same with `setuptools`.

Reply:
@rarkins WDYT?

Reply:
Does the git blame give any hint?

Reply:
The setup.cfg didn't have a huge history. Maybe @zharinov can tell us more?

Reply:
OK I don't see any justification for it..